test/truffle/ecosystem/rails-app.sh

```diff
@@ -5,9 +5,7 @@ set -x
 
 cd ../jruby-truffle-gem-test-pack/gem-testing/rails-app
 
-JRUBY_BIN=../../../jruby/bin
-JRUBY=$JRUBY_BIN/jruby
-JTR=$JRUBY_BIN/jruby-truffle-tool
+JTR=../../../truffleruby/bin/jruby-truffle-tool
```

Comment:
It would be more robust to get the root of the repo via:
```bash
root=`pwd` # before the cd
```
since it's assumed above anyway that the script is run from the repo root.